vortexasdk/client.py

```diff
@@ -95,7 +98,14 @@ def _process_multiple_pages(
                 return pool.map(func, offsets)
 
     @staticmethod
-    def _calculate_total(response) -> int:
+    def _cleanse_payload(payload: Dict) -> Dict:
+        filtered = {
```

Comment:
We filter this payload.

It's also sorted for consistency when reading the logs.